app-backend/build.sbt

```diff
@@ -150,6 +151,8 @@ lazy val common = Project("common", file("common"))
   .settings({libraryDependencies ++= testDependencies ++ Seq(
     Dependencies.jwtCore,
```

Comment:
Should we be removing these dependencies now that we're using another library for JWTs?

Reply:
Yeah -- there is some clean up that needs to happen on this branch